pythran/pythonic/types/numpy_texpr.hpp

```diff
@@ -200,5 +200,15 @@ struct __combined<pythonic::types::numpy_texpr<E>, K> {
     typedef pythonic::types::numpy_texpr<E> type;
 };
 
+template<class E, class T, size_t N>
+struct __combined<pythonic::types::numpy_texpr<E>, pythonic::types::ndarray<T, N>> {
+    typedef pythonic::types::ndarray<T, N> type;
+};
+
+template<class E, class T, size_t N>
+struct __combined<pythonic::types::ndarray<T, N>, pythonic::types::numpy_texpr<E>> {
+    typedef pythonic::types::ndarray<T, N> type;
+};
```

Comment:
I don't get the issue. If it is a strange combine, I evaluate in and save it in a ndarray. Why it is incorrect? (Pythran compiled code works with these cases).